db/tasks.php

```diff
@@ -107,5 +107,14 @@
         'dayofweek' => '*',
         'month'     => '*'
     ),
+        array(
+        'classname' => 'tool_objectfs\task\delay_delete_external_objects',
+        'blocking'  => 0,
+        'minute'    => '*',
+        'hour'      => '*',
```

Comment:
I think we can run this in the same manner as the `tool_objectfs\task\orphan_objects` and the `tool_objectfs\task\delete_orphaned_object_metadata` tasks. This is optional, just an opinion.
```suggestion
        'minute'    => 'R',
        'hour'      => 'R',
```